src/DetailsView/components/automated-checks-command-bar.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { FeatureFlags } from 'common/feature-flags';
+import { NamedFC } from 'common/react/named-fc';
+import { ReportExportComponentProps } from 'DetailsView/components/report-export-component';
+import { getReportExportComponentPropsForAutomatedChecks } from 'DetailsView/components/report-export-component-props-factory';
+import { CommandBarProps, ReportExportComponentPropertyConverter } from './details-view-command-bar';
+import { DetailsViewCommandBar } from './details-view-command-bar';
+
+export const AutomatedChecksCommandBar = NamedFC<CommandBarProps>('AutomatedChecksCommandBar', props => {
+    const converter: ReportExportComponentPropertyConverter = props.reportExportComponentPropertyFactory;
+    let reportExportComponentProps: ReportExportComponentProps;
+    if (converter) {
```

Comment:
I'm not fully following why this if-block is necessary in either command bars: when would the ```reportExportComponentPropertyFactory``` not exist?

Reply:
The override was added primarily for testing, so that I could test just the command bar in isolation from the factory method. In actual usage, the factory property is not being set. I'll try to move it into the deps